tests/dtm/test_single_process_restart.py

```diff
@@ -171,3 +178,122 @@ def test_write_during_m0d_restart(self):
 
         self.test_completed = True
         self.log.info("ENDED: Verify READ during m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41232")
+    def test_copy_object_after_m0d_restart(self):
+        """Verify copy object after m0d restart using pkill."""
+        self.log.info("STARTED: Verify copy object after m0d restart using pkill")
+        bucket_name_1 = 'bucket-test-41232-1'
+        bucket_name_2 = 'bucket-test-41232-2'
+        object_name = 'object-test-41232'
+        self.log.info("Step 1: Start write Operations :")
+        resp = self.s3_obj.create_bucket(bucket_name=bucket_name_1)
+        assert_utils.assert_true(resp[0], resp[1])
+        resp = self.s3_obj.create_bucket(bucket_name=bucket_name_2)
+        assert_utils.assert_true(resp[0], resp[1])
+        for size in self.test_cfg["size_list"]:
+            file_name = "{}{}".format("dtm-test-41232", size)
+            file_path = os.path.join(self.dir_path, file_name)
+            system_utils.create_file(file_path, size)
+            resp = self.s3_obj.put_object(bucket_name_1, f"{object_name}_{size}", file_path)
+            assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Step 2 : Perform Single m0d Process Restart During Write Operations")
+        self.dtm_obj.process_restart(self.master_node_list[0],
+                                     POD_NAME_PREFIX, MOTR_CONTAINER_PREFIX, self.m0d_process)
+        self.log.info("Step 3: Check hctl status if all services are online")
```

Comment:
hctl status check is not required, its taken care in process restart itself. 
Please rebase the branch and make necessary changes.

Reply:
fixed